### PR TITLE
[RFR] Add notifications

### PR DESCRIPTION
**Pull request by manuquentin**
capture d ecran 2015-06-15 a 17 19 39

### CODE REVIEW

**Comment by fzaninotto** [comment]
Is ot the same look and feel as in ng-admin ?

**Comment by manuquentin** [comment]
Yes, I've added a picture

**Comment by manuquentin** [comment]
Back to WIP, missing tests

File: e2e/creationViewSpec.js [hunk 1]
```diff
@@ -17,6 +17,12 @@ describe('CreationView', function () {
         it('should update values on form submit and redirect to edit page', function () {
             $('#create-view .react-admin-field-title input').sendKeys('My new post').then(function () {
                 $('#create-view button[type="submit"]').click().then(function () {
+                    // Wait for notification to display
+                    browser.driver.sleep(1000);
```
Comment (jeromemacias) [inline e2e/creationViewSpec.js]: You should use `browser.wait()`.

File: e2e/deleteView.js [hunk 2]
```diff
@@ -26,7 +26,12 @@ describe('DeleteView', function () {
 
                 // Delete it
                 $('.btn-danger').click().then(function () {
-                    // Wait for list to load
+                    // Wait for notification to display
+                    browser.driver.sleep(1000);
```
Comment (jeromemacias) [inline e2e/deleteView.js]: Same here.

File: e2e/editionViewSpec.js [hunk 3]
```diff
@@ -17,7 +17,13 @@ describe('EditionView', function () {
         it('should update values on form submit', function () {
             $('#edit-view .react-admin-field-title input').sendKeys(' and what ?').then(function () {
                 $('#edit-view button[type="submit"]').click().then(function () {
-                     expect($('.page-header h1').getText()).toContain('and what ?');
+                    // Wait for notification to display
+                    browser.driver.sleep(1000);
```
Comment (jeromemacias) [inline e2e/editionViewSpec.js]: Same here.

File: examples/blog/index.html [hunk 4]
```diff
@@ -5,6 +5,8 @@
     <title>React-Admin Blog Sample Application</title>
 </head>
 <body>
+    <div id="my-app"></div>
```
Comment (jeromemacias) [inline examples/blog/index.html]: How it worked before ?

Comment (manuquentin) [inline examples/blog/index.html]: The app was renderer un the `body` but since notifications are also in the body, react delete them automatically.

**Comment by jeromemacias** [comment]
Is it RFR ?

**Pull request merged**